packages/now-cli/src/commands/deploy/latest.js

```diff
@@ -435,6 +439,19 @@ export default async function main(
     }
   }
 
+  const rootDirectory =
+    (project ? project.rootDirectory : await inputRootDirectory(autoConfirm)) ||
+    null;
```

Comment:
I think we should move this inside the `if (status === 'not_linked')` condition.

Reply:
Moved it